src/main/java/uk/gov/hmcts/divorce/citizen/event/PaymentMade.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.divorce.citizen.event;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.divorce.citizen.notification.ApplicationSubmittedNotification;
+import uk.gov.hmcts.divorce.common.model.CaseData;
+import uk.gov.hmcts.divorce.common.model.State;
+import uk.gov.hmcts.divorce.common.model.UserRole;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+
+import java.util.Set;
+
+import static uk.gov.hmcts.divorce.common.model.State.Draft;
+import static uk.gov.hmcts.divorce.common.model.State.Submitted;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_COURTADMIN;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CASEWORKER_DIVORCE_COURTADMIN_BETA;
+import static uk.gov.hmcts.divorce.common.model.UserRole.CITIZEN;
+import static uk.gov.hmcts.divorce.common.model.access.Permissions.CREATE_READ_UPDATE;
+
+@Component
+public class PaymentMade implements CCDConfig<CaseData, State, UserRole> {
+
+    public static final String PAYMENT_MADE = "payment-made";
+
+    @Autowired
+    private ApplicationSubmittedNotification notification;
+
+    @Override
+    public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        configBuilder
+            .event(PAYMENT_MADE)
+            .initialState(Draft)
+            .postState(Set.of(Submitted))
```

Comment:
do you want to change the state by default to submitted or do you want to leave this to * and then in the callback change it to submitted or awaiting payment depending on the outcome ?

Reply:
Yes that would be more sensible